src/lib/bootloader/proposal_client.rb

```diff
@@ -76,6 +77,18 @@ def make_proposal(attrs)
     end
 
     def ask_user(param)
+      if Yast::Mode.update
+        current_bl = ::Bootloader::BootloaderFactory.current
```

Comment:
NP: I know that this is a backport, but I must to say that I prefere `current_bootlader` as param name. More meaningful, IMO :angel:

Reply:
it is just two line scope. So for me it was fine. I picking better names if it has longer scope. So single or two letter for single line is fine and then increasing it.